src/persist/src/storage.rs

```diff
@@ -60,10 +60,6 @@ pub trait Buffer {
 ///
 /// - Invariant: Implementations are responsible for ensuring that they are
 ///   exclusive writers to this location.
-///
-/// TODO: Document restrictions on what keys are legal.
```

Comment:
Moved this TODO to a list I'm starting of things to circle back on closer to 1.0